assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -71,9 +71,12 @@ schedule.core_pool_size=10
 machine.docker.registry=${CHE_REGISTRY_HOST}:5000
 machine.docker.snapshot.registry_namespace=NULL
 machine.docker.unused_containers_cleanup_period_min=60
-docker.registry.auth.url=http://${CHE_REGISTRY_HOST}:5000
-docker.registry.auth.username=NULL
-docker.registry.auth.password=NULL
+
+### Docker registry auth config example
+#docker.registry.auth.your_registry_name.url=https://index.docker.io/v1/
+#docker.registry.auth.your_registry_name.username=user-name
+#docker.registry.auth.your_registry_name.password=user-password
```

Comment:
It will be good to add comment which says that user can configure many registries with different names, because it is not obviously now.